src/buildCommon.js

```diff
@@ -234,7 +235,11 @@ const makeOrd = function(
     // Math mode or Old font (i.e. \rm)
     const isFont = mode === "math" || (mode === "text" && options.font);
     const fontOrFamily = isFont ? options.font : options.fontFamily;
-    if (fontOrFamily) {
+    if (value.charCodeAt(0) === 0xD835) {
+        // surrogate pairs get special treatment
+        const [wideFontName, wideFontClass] = wideCharacterFont(value, mode);
```

Comment:
In the future it would be nice if we had a function that handled the non-wide character situation in a similar way to how `wideCharacterFont` does it.